### DISCUSSION TITLE
doesn't work with SolidStart in prod mode

### DISCUSSION THREAD

**Discussion by dn1t** [vanilla-extract-css/vanilla-extract#1434] [Q&A]
Hi, I'm trying to use vanilla-extract with [SolidStart](https://github.com/solidjs/solid-start). It works flawlessly in dev mode, but when I build the project, start it with `vinxi start`, and navigate to localhost:3000, the page doesn't load and prints the following error on the console.

```
[nitro] [unhandledRejection] TypeError: Cannot read properties of undefined (reading 'file')
    at Object.get (file:///workspaces/tuned/.output/server/chunks/runtime.mjs:5598:33)
    at file:///workspaces/tuned/.output/server/chunks/runtime.mjs:6689:64
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```

I think it's an error from [nitro](https://github.com/unjs/nitro)(according to the logs) used in SolidStart, but I couldn't find any info on the whole internet including nitro's nor solidstart's github issues, so I just decided to leave a question here.

The whole code is [here](https://github.com/dn1t/tuned).

**Comment by askoufis** [comment]
The repo you've linked doesn't have any Vanilla Extract styles in it. If you can make a minimal reproduction of the bug, feel free to raise an issue so we can track it.

**Reply by dn1t** [reply]
I accidentally deleted the commit with vanilla-extract. It was just a new solidstart project generated with create-solid and added `vanillaExtractPlugin` in vite.plugins in `app.config.ts`.
